Dockerfile

```diff
@@ -1,26 +1,17 @@
-FROM debian:sid
+FROM openjdk:8-jre-alpine
 
 MAINTAINER Christian Kuntzsch <christian@kuntzsch.me>
 
-RUN \
-    apt-get update && \
-    apt-get install -y openjdk-8-jre wget
-
-ENV JAVA_HOME /usr/lib/jvm/java-8-openjdk-amd64
-
-RUN \
-    mkdir -p /var/otp && \
```

Comment:
Register the environment variable before and then use it as target for the download.